yarn-project/circuits.js/src/hash/hash.ts

```diff
@@ -28,18 +28,18 @@ export function computeNoteHashNonce(nullifierZero: Fr, noteHashIndex: number):
  * Computes a siloed note hash, given the contract address and the note hash itself.
  * A siloed note hash effectively namespaces a note hash to a specific contract.
  * @param contract - The contract address
- * @param uniqueNoteHash - The unique note hash to silo.
+ * @param noteHash - The note hash to silo.
  * @returns A siloed note hash.
  */
-export function siloNoteHash(contract: AztecAddress, uniqueNoteHash: Fr): Promise<Fr> {
-  return poseidon2HashWithSeparator([contract, uniqueNoteHash], GeneratorIndex.SILOED_NOTE_HASH);
+export function siloNoteHash(contract: AztecAddress, noteHash: Fr): Promise<Fr> {
```

Comment:
There's an ugly naming discrepency between `siloNoteHash` and `computeUniqueNoteHash` below. Maybe both should start with `compute...`?
(Clearly you didn't introduce this, but I'm flagging as I see it for the first time)

Reply:
Yea didn't rename those as that'd create a bunch of unrelated entries in the diff updating the callsites